components/global/Carousel/index.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import Carousel from ".";
+
+describe("<Carousel/>", () => {
+  test("Carousel", () => {
+    expect(() => {
+      render(<Carousel />);
+    }).not.toThrowError();
```

Comment:
Add a snapshot test

Reply:
I have added content based test, but using testing-library, it is a little hard to do snapshot test, because the class name is randomly generated when style-components are being used, because I'm unaware of what they would be. I believe storybook-testing might be better and might provide us with snapshot testing for our scenario, Please advice.